astro/src/diagrams/docs/get-started/core-concepts/integration-points/login-api-flow-federation.astro

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
Pretty cool how a sequence diagram can be created.

Reply:
100%